bin/codebasin

```diff
@@ -9,35 +9,13 @@ import argparse
 import logging
 import os
 import sys
-import warnings
 
 from codebasin import config, finder, report, util
 from codebasin.walkers.platform_mapper import PlatformMapper
 
 version = "1.2.0"
```

Comment:
I think we should leave it for later.

Our next release has to be 2.0.0 (because we've broken backwards compatibility now), but the first breaking change isn't actually this PR -- it was in one of the other removals of deprecated functionality.  There's not going to be any need for us to bump the version before our next release (i.e., a 2.1.0 won't make sense before 2.0.0) so we may as well wait.